simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,865 @@
+    at a given moment in time.
+    """
+
+    name = Quantity(
+        type=MEnum("original", "primitive", "standard"),
+        description="""
+        Name to identify the cell structure. It might be:
+            - 'original' as in orignally parsed,
+            - 'primitive' as the primitive unit cell,
+            - 'standard' as the standarized cell used for referencing.
+        """,
+    )
+
+    labels = Quantity(
+        type=str,
+        shape=["*"],
+        description="""
+        List containing the labels of the atomic species in the system at the different positions
+        of the structure. It refers to a chemical element as defined in the periodic table,
+        e.g., 'H', 'O', 'Pt'. This quantity is equivalent to `atomic_numbers`.
+        """,
+    )
+
+    atomic_numbers = Quantity(
+        type=np.int32,
+        shape=["*"],
+        description="""
+        List of atomic numbers Z. This quantity is equivalent to `labels`.
+        """,
+    )
+
+    positions = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter",
+        description="""
+        Positions of all the atoms in Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="meter",
+        description="""
+        Lattice vectors of the simulated cell in Cartesian coordinates. The first index runs
+        over each lattice vector. The second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors_reciprocal = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="1/meter",
+        description="""
+        Reciprocal lattice vectors of the simulated cell, in Cartesian coordinates and
+        including the $2 pi$ pre-factor. The first index runs over each lattice vector. The
+        second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    periodic_boundary_conditions = Quantity(
+        type=bool,
+        shape=[3],
+        description="""
+        If periodic boundary conditions are applied to each direction of the crystal axes.
+        """,
+    )
+
+    velocities = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter / second",
+        description="""
+        Velocities of the atoms. It is the change in cartesian coordinates of the atom position
+        with time.
+        """,
+    )
+
+    supercell_matrix = Quantity(
+        type=np.int32,
+        shape=[3, 3],
+        description="""
+        Specifies the matrix that transforms the primitive unit cell into the supercell in
+        which the actual calculation is performed. In the easiest example, it is a diagonal
+        matrix whose elements multiply the `lattice_vectors`, e.g., [[3, 0, 0], [0, 3, 0], [0, 0, 3]]
+        is a $3 x 3 x 3$ superlattice.
+        """,
+    )
```

Comment:
Ok, I see your point in `name` now.

I think one could decide to store this matrix as a way of defining their supercells, as an addition to positions and lattice_vectors. This is a very typical way of defining supercells.

This could also help in normalizing back from a given primitive cell into the original one, as maybe there is a way of finding the matrix. Maybe we should talk with Lauri about this.

Reply:
> I think one could decide to store this matrix as a way of defining their supercells, as an addition to positions and lattice_vectors. This is a very typical way of defining supercells.

Indeed, but then we already need to have a primitive / conventional cell to be defined.